### PR TITLE
Main panel

### PR DESCRIPTION
**Pull request by ColeMaddison**

### CODE REVIEW

File: lohica-js-network/src/components/loginForm/LoginEmailComponent.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ class LoginEmailComponent extends React.Component {
     constructor(props){
         super(props);
 
-        this.state = {
-            show: false
-        }
+        // this.state = {
```
Comment (mbielkin-virtru) [inline lohica-js-network/src/components/loginForm/LoginEmailComponent.js]: Remove all the comments from the PR if you do not need it.

File: lohica-js-network/src/components/userAccountComponents/UserAccountComponent.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+import { EmailInput, GenderRadio, AgeInput, ImageInput, NameInput, SurnameInput, MidnameInput } from '../index';
 
 class UserAccountComponent extends React.Component{
+
+    constructor(props){
+        super(props);
+
+        console.log(this.props);
+
+        fetch('/account', {
```
Comment (mbielkin-virtru) [inline lohica-js-network/src/components/userAccountComponents/UserAccountComponent.js]: Use constants for all endpoints url

File: server.js [hunk 3]
```diff
@@ -40,13 +40,22 @@ app.post(routes.loginRoute, (req, res) => {
...
+});
+
 // route to check the token for validity
 app.get(routes.checkTokenRoute, (req, res) => {
-    console.log(req.headers);
     if(req.headers){
         jwt.verify(req.headers.authorization.split(' ')[1], secretConfig.secret, function(err, decoded){
-            console.log(decoded);
-            if(err){
+            let date = new Date().getTime();
```
Comment (mbielkin-virtru) [inline server.js]: use const except of let if you do not change date later

**Comment by mbielkin-virtru** [comment]
Merge this one.

File: lohica-js-network/src/components/loginForm/LoginEmailComponent.js (hunk 1 above)
Comment (ColeMaddison) [reply lohica-js-network/src/components/loginForm/LoginEmailComponent.js]: done

File: lohica-js-network/src/components/userAccountComponents/UserAccountComponent.js (hunk 2 above)
Comment (ColeMaddison) [reply lohica-js-network/src/components/userAccountComponents/UserAccountComponent.js]: done

File: server.js (hunk 3 above)
Comment (ColeMaddison) [reply server.js]: done

**Comment by mbielkin-virtru** [comment]
Let's merge this branch? Here are 50+ files and it becomes difficult to do a review... @gavrilenco any concerns?

**Comment by gavrilenco** [comment]
@mbielkin-virtru makes sense, merge it

**Pull request merged**